src/api-v3/paths/preparations/payments.spec.ts

```diff
@@ -0,0 +1,126 @@
+// Unit tests for:
+//     GET /v3/preparations/payments
+
+import sinon from 'sinon';
+import request from 'supertest';
+import { mockApp, rippleApi } from "../../../fixtures/mocks";
+import { expect } from 'chai';
+
+const path = '/v3/preparations/payments';
+
+describe(path, () => {
+  it('prepares a payment - normal happy path', (done) => {
+    sinon.stub(rippleApi.connection, 'getReserveBase').resolves(20000000);
+    sinon.stub(rippleApi, 'request').withArgs('account_info', sinon.match.any).resolves({
+      account_data: {
+        Balance: "20000000",
+        Flags: 0
+      }
+    });
+    sinon.stub(rippleApi, 'getLedgerVersion').resolves(7000000);
+    sinon.stub(rippleApi, 'prepareTransaction').withArgs(sinon.match.any, {}).resolves({
+      txJSON: JSON.stringify({
+        "TransactionType": "Payment",
+        "Account": "rNQao3Z1irwRjKWSs8heL4a8WKLPKfLrXs",
+        "Destination": "rpgHWJdXkSvvzikdJCpuMzU7zWnuqsJRCZ",
+        "Amount": "123000000",
+        "Flags": 2147483648,
+        "LastLedgerSequence": 7000003,
+        "Fee": "12",
+        "Sequence": 7
+      }),
+      instructions: {
+        fee: "0.000012",
+        sequence: 7
+      }
+    });
+
+    // GIVEN a source, destination, and amount to send
+    const query = {
+      source: 'XVDDpvwxP3yZaZGU1Gik6HxJ4kkAEy6roNBwRFT3eRecYmH',
+      destination: 'X7ZrF85JfL7AwEg7MTwHuWNJjCrPrhHL3bPGHU9UASVz4eX',
+      value: '123',
+      currency: 'XRP'
+    };
+
+    // WHEN preparing a payment
+    request(mockApp)
+      .get(path)
+      .query(query)
+      .expect(200)
+      // THEN return a payment
+      .expect({
+        "TransactionType": "Payment",
+        "Account": "rNQao3Z1irwRjKWSs8heL4a8WKLPKfLrXs",
+        "Destination": "rpgHWJdXkSvvzikdJCpuMzU7zWnuqsJRCZ",
+        "Amount": "123000000",
+        "Flags": 2147483648,
+        "LastLedgerSequence": 7000003,
+        "Fee": "12",
+        "Sequence": 7,
+        "min_ledger": 7000000,
+        "max_ledger": 7000003
+      })
+      .end(done);
+  });
+
+  it('prepares a payment with a custom maxLedgerVersionOffset', (done) => {
+    sinon.stub(rippleApi.connection, 'getReserveBase').resolves(20000000);
```

Comment:
Optional: Stylistically, I'd write this all under `GIVEN` as `GIVEN mocked networking and a max ledger version offset`. 

That way it's clear what sinon is up to. You can be more specific if you want too. 

If you feel it's really complicated, then you could write something line:
```
// GIVEN mocked networking which will ....
sinon.stub(..)
...

// AND a maxLedgerVersionOffset
const maxLedgerVersionOffset = 500;
```